API/Controllers/TachiyomiController.cs

```diff
@@ -21,14 +16,12 @@ namespace API.Controllers;
 public class TachiyomiController : BaseApiController
 {
     private readonly IUnitOfWork _unitOfWork;
-    private readonly IReaderService _readerService;
-    private readonly IMapper _mapper;
+    private readonly ITachiyomiService _tachiyomiService;
 
-    public TachiyomiController(IUnitOfWork unitOfWork, IReaderService readerService, IMapper mapper)
+    public TachiyomiController(IUnitOfWork unitOfWork, ITachiyomiService tachiyomiService, IMapper mapper)
```

Comment:
I still see Mapper injected, but it's not needed anymore

Reply:
That's weird. I don't see it on mine